packages/tasks/src/model-libraries.ts

```diff
@@ -116,6 +116,15 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoName: "doctr",
 		repoUrl: "https://github.com/mindee/doctr",
 	},
+	elm: {
+		prettyLabel: "ELM",
+		repoName: "elm",
+		repoUrl: "https://github.com/slicex-ai/elm-v0.1",
+		filter: false,
+		countDownloads: {
+			terms: { path: ["slicex_elm_config.json"] },
```

Comment:
Hi @chinnadhuraisankar, thanks for opening a PR! I noticed the ELM models weights are all hosted in the same repo. I would recommend to split your [slicexai/elm-v0.1](https://huggingface.co/slicexai/elm-v0.1) model repo into 12 different repos (`elm-0.25_news_classification`, `elm-0.75_news_classification`, `elm-1.0_news_content_generation`, etc.) which would allow you to have a download count per model and not just for the whole collection. If you want, you can programmatically manage those repos using the `huggingface_hub` Python library (see [docs](https://huggingface.co/docs/huggingface_hub/guides/overview)). 

Also, I noticed some folders have a `slicex_elm_config.json` file ([example](https://huggingface.co/slicexai/elm-v0.1/tree/main/models/elm-1.0_news_content_generation)) but some don't ([example](https://huggingface.co/slicexai/elm-v0.1/tree/main/models/elm-1.0_news_content_generation)). To correctly track downloads you should split them in multiple repos and then add this file to each of them.

Besides that, once you have multiple repos, you can edit the model cards to include more information and in particular:
- `library_name` (set to `elm`?) => this way all models will be easily discoverable + it would activate the download count
- `pipeline_tag` (set to `text-classification`/`summarization`?) => helps for referencing on the hf.co/models page
For more details on Model Card metadata, check out [this guide](https://huggingface.co/docs/hub/model-cards#model-card-metadata). 

Finally, I would recommend to create [Collection](https://huggingface.co/docs/hub/collections) of all your models as a "ELM v0.1 collection" for instance so that all repos are grouped together for this release. In the future you could have new collections ("ELM v0.2", etc.). 

What do you think? :)

Reply:
Hi @Wauplin, 

1. Thanks your helpful suggestions. We already organized into 4 task specific repos in private mode (one per use-case - because `elm-0.75` & `elm-0.25` are different slices derived from the same `elm-1.0` checkpoint 
     - `elm-v0.1_toxicity_generation`, `elm-v0.1_news_summarization`, `elm-v0.1_news_content_generation` and `elm-v0.1_news_classification`.  
3. If we place the `slicex_elm_config.json` within each checkpoint and 3 checkpoints per repo, would the download count still be considered 1 per repo or 3 per repo?
4. Thanks for the suggestion regarding the pipeline tag, we have now added appropriate tags to each repo. 
5. Thanks for collections recommendation. We are in the process of creating one.

Since we are planning to open-source the repos asap,  it would be really helpful if you could approve this PR so that we could make track model download via the library tag in a timely manner!

Reply:
Additionally, 

5. We also noticed that usage statistics/analytics doesn't seem to update for our public space. Is it usually delayed?
6. Does the download stats track "config.json" even for non transformer models?

Thanks a lot!

Reply:
> If we place the slicex_elm_config.json within each checkpoint and 3 checkpoints per repo, would the download count still be considered 1 per repo or 3 per repo?

I'm not sure to fully understand the repo structure here. If you end up with 3 `slicex_elm_config.json` files in the repo, then the download count will be a cumulated sum of the downloads of the 3 files. This is why it is usually recommended to track downloads on a single file per repo. 

> We also noticed that usage statistics/analytics doesn't seem to update for our public space. Is it usually delayed?

Analytics are not live and you might need to wait for a few hours/days to get the exact numbers. 

> Does the download stats track config.json even for non transformer models?

Yes by default `config.json` downloads are counted. But as soon as you define new rules, the default rules won't apply anymore. See https://huggingface.co/docs/hub/models-download-stats#which-are-the-query-files-for-different-libraries. Also, please note that custom rules are not retroactive.

Reply:
Btw, to confirm but `[slicex_elm_config.json](terms: { path: ["slicex_elm_config.json"])` won't count configs in subfolders. You should update to a wildcard if you want to do that:

```js
countDownloads: { wildcard: { path: "*/slicex_elm_config.json" } },
```

(see [whispetkit](https://github.com/huggingface/huggingface.js/blob/725997c1693c1cb899e5f645b8f8a5ef9f535284/packages/tasks/src/model-libraries.ts#L432) for instance)